src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -32,34 +30,52 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setParameter('sulu_media.search.default_image_format', $config['search']['default_image_format']);
+        // format manager
         $container->setParameter('sulu_media.format_manager.response_headers', $config['format_manager']['response_headers']);
         $container->setParameter(
             'sulu_media.format_manager.default_imagine_options',
             $config['format_manager']['default_imagine_options']
         );
+        $container->setParameter('sulu_media.format_manager.config_paths', $config['format_manager']['config_paths']);
+        $container->setParameter('sulu_media.format_manager.mime_types', $config['format_manager']['mime_types']);
+
+        // format cache
+        $container->setParameter('sulu_media.format_cache.path', $config['format_cache']['path']);
+        $container->setParameter('sulu_media.format_cache.save_image', $config['format_cache']['save_image']);
+        $container->setParameter('sulu_media.format_cache.segments', $config['format_cache']['segments']);
+
+        // converter
+        $container->setParameter('sulu_media.image.command.prefix', 'image.converter.prefix.');
```

Comment:
Not related to this PR, but should be solved somehow using the dependency injection container tags.